services/core-api/app/api/mines/tailings/resources/tailings.py

```diff
@@ -53,6 +53,12 @@ class MineTailingsStorageFacilityResource(Resource, UserMixin):
         help='GUID of the party that is the Engineer of Record for this TSF.',
         location='json',
         store_missing=False)
+    parser.add_argument(
+        'notes',
```

Comment:
Does this not need to be added to the put?

Reply:
Thanks for the review. Yup, this one is required for modal create/update.